libraries/SITL/SIM_Precland.h

```diff
@@ -37,10 +37,15 @@ class SIM_Precland {
     uint32_t last_update_ms() const { return _last_update_ms; }
 
     const Vector3d &get_target_position() const { return _target_pos; }
-    bool is_enabled() const {return static_cast<bool>(_enable);}
+     bool is_enabled() const {return static_cast<bool>(_enable);}
```

Comment:
I guess a little bit of white space has been accidentally added to this line.

Reply:
Oops. Yeah. Sorry for that. I have fixed it/